app/views/consent_forms/confirm.html.erb

```diff
@@ -0,0 +1,12 @@
+<% content_for :before_main do %>
+  <%= render AppBacklinkComponent.new(
+    href: url_for(action: :start),
+    name: "start page",
+  ) %>
+<% end %>
```

Comment:
One thing we've been bad at doing is adding different page titles, which is good for accessibility:

```suggestion
<% end %>

<%= content_for :page_title, "Check and confirm answers" %>
```

Reply:
As discussed, this will be addressed in a separate PR.